jina/peapods/runtimes/gateway/prefetch.py

```diff
@@ -46,16 +81,16 @@ async def prefetch_req(num_req, fetch_to):
                         raise TypeError(
                             f'{typename(request_iterator)} does not have `__anext__` or `__next__`'
                         )
+
+                    future = Future()
```

Comment:
```suggestion
                    future = get_or_reuse_loop().create_future()
```
[Official docs](https://docs.python.org/3/library/asyncio-future.html#asyncio.Future) recommend using `loop.create_future()`. Not important right notw but it might be required as we use uvloop.